src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveImmBinaryOpTestTemplate.template

```diff
@@ -233,7 +255,7 @@ namespace JIT.HardwareIntrinsics.Arm
                                      .Invoke(null, new object[] {
                                         Unsafe.Read<{Op1VectorType}<{Op1BaseType}>>(_dataTable.inArray1Ptr),
                                         Unsafe.Read<{Op1VectorType}<{Op1BaseType}>>(_dataTable.inArray2Ptr),
-                                        (byte){Imm}
+                                        (byte)Imm
```

Comment:
By having `Imm`, we will always generate table driven logic, while with `{Imm}`, we will always generate the single instruction, with constant embedded. I think we need to have a test for both and I have captured it in https://github.com/dotnet/runtime/issues/104809. Please revert these changes and we will address them together as part of that issue.

Reply:
Got it, I did this out of convenience for writing the test definitions: If we do `{Imm}`, then we cannot dynamically generate the immediate value using something like `TestLibrary.Generator.GetByte()` because the value will change each time we use `{Imm}`. Instead, we'll have to hard-code the immediate into the test with something like `["Imm"] = "0"`. I don't mind doing this; it will just result in more tests for this API.